internify/src/components/pages/CreateJobPosting/CreateJobHeader.js

```diff
@@ -6,13 +6,15 @@ import {
   LengthSubHeader,
   Stepper,
 } from "../../molecules/index";
+import AuthNavBar from "../../organisms/AuthNavbar";
 import { ButtonFilled } from "../../atoms/Button";
 import "./styles/CreateJobHeader.css";
 
 function CreateJobHeader(props) {
   return (
     <div className="create">
       <Container maxWidth="md" className={"container"}>
+        <AuthNavBar></AuthNavBar>
```

Comment:
Shorthand version is:
`<AuthNavBar/>`

Also, change the name of `NavBar` to `Navbar` like I did in the recent refactoring work

Reply:
Noted - will fix!